pallets/streaming/src/lib.rs

```diff
@@ -51,6 +51,8 @@ pub use pallet::*;
 pub mod weights;
 pub use weights::WeightInfo;
 
+pub const MINIMUM_DEPOSIT_AMOUNT: u128 = 10_000_000_000_000_000_000_000;
```

Comment:
I guess this will not work for some tokens with decimals 6. We should only allow to create stream if the minimumDeposit is set I guess

Reply:
Okay, it does make it better